code_challenges/code_wars/even_or_odd.py

```diff
@@ -0,0 +1,8 @@
+# https://www.codewars.com/kata/53da3dbb4a5168369a0000fe
+
+
+def even_or_odd(number):
+    if number % 2 == 0:
+        return "Even"
+    else:
+        return "Odd"
```

Comment:
Since the `else` is a catch all this could be simplified to:
```python
def even_or_odd(number):
    if number % 2 == 0:
        return "Even"
    return "Odd"
```